test/jdk/java/security/MessageDigest/TestCloneable.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file, including the copyright date and the final newline, should be identical to jdk17.

Reply:
Updated

Reply:
The copyright date should be 2020; see https://github.com/ibmruntimes/openj9-openjdk-jdk17/blob/openj9/test/jdk/java/security/Signature/TestCloneable.java#L2.